FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c

```diff
@@ -100,33 +93,36 @@ MetricsCollectorStatus_t xGetOpenTcpPorts( uint16_t * pusOutTcpPortsArray,
                                            uint32_t ulTcpPortsArrayLength,
                                            uint32_t * pulOutNumTcpOpenPorts )
 {
-    MetricsCollectorStatus_t status = MetricsCollectorSuccess;
+    MetricsCollectorStatus_t eStatus = MetricsCollectorSuccess;
```

Comment:
Doesn't the variable name need to be xStatus as per FreeRTOS coding standards?

Reply:
Its an enum. Fixed name of enum typedef and options.